pandas-iris/{{ cookiecutter.repo_name }}/src/tests/test_run.py

```diff
@@ -21,13 +22,14 @@ def config_loader():
 
 
 @pytest.fixture
-def project_context(config_loader):
-    return KedroContext(
-        package_name="{{ cookiecutter.python_package }}",
-        project_path=Path.cwd(),
-        config_loader=config_loader,
-        hook_manager=_create_hook_manager(),
-    )
+def project_context():
+    def project_context(config_loader):
+        return KedroContext(
+            package_name="sajd",
```

Comment:
package_name="{{ cookiecutter.python_package }}",

Reply:
Technical question 😅 How do you make comments about code on Github PR so people can apply the change directly with the button?
@AntonyMilneQB

Reply:
https://til.hashrocket.com/posts/szgdi4at59-github-insert-a-suggestion-

It works on multiple lines now as well. You can add suggestions to a batch and then commit several in one go. Super handy feature.

Reply:
Yes, this is very handy! And I like that it is simply a markdown block!

Reply:
Oops that wasn’t supposed to replace that. Thank you!